### PR TITLE
[Complete] Even more linting fixes

### PR DESCRIPTION
**Pull request by andylolz**
Last one of these from me, I think!

This gets `make pydocstyle` passing, and brings `make pylint` to 9.36 / 10.

Remaining stuff is covered in other pull requests (#203; #205).

### CODE REVIEW

**Comment by andylolz** [comment]
```
$ make pylint
Your code has been rated at 8.69/10 (previous run: 8.14/10, +0.55)
```

File: iati/about/models.py [hunk 1]
```diff
@@ -9,7 +11,7 @@
 from home.models import AbstractContentPage, AbstractIndexPage, DefaultPageHeaderImageMixin, PullQuoteBlock
 
 
-class AboutPage(DefaultPageHeaderImageMixin, AbstractContentPage):
+class AboutPage(DefaultPageHeaderImageMixin, AbstractContentPage):  # pylint: disable=too-many-ancestors
```
Comment (allthatilk) [inline iati/about/models.py]: As this ancestors linter fix is so prolific, it's probably best to specify it at a higher level in `setup.cfg`

File: iati/about/models.py [hunk 2]
```diff
@@ -64,18 +66,15 @@ def get_context(self, request, *args, **kwargs):
...
     parent_page_types = ['about.CaseStudyIndexPage']
     subpage_types = []
 
     feed_image = models.ForeignKey(
-        'wagtailimages.Image',
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+',
+        'wagtailimages.Image', null=True, blank=True,
```
Comment (allthatilk) [inline iati/about/models.py]: As the previous state for this looked cleaner and was slightly more readable, we'd prefer to mark this for the linters to ignore over the changes made here.

Comment (allthatilk) [reply iati/about/models.py]: This applies to the same changes elsewhere too.

Comment (andylolz) [reply iati/about/models.py]: > we'd prefer to mark this for the linters to ignore

^^ I should have explained – I think it’s only possible to do this globally. I don’t think it’s possible to ignore `duplicate-code` errors case by case.

Comment (allthatilk) [reply iati/about/models.py]: For reference: https://github.com/PyCQA/pylint/issues/214

Gah! That's annoying. Will accept this change then as there don't seem to be any better options.

Comment (andylolz) [reply iati/about/models.py]: Thanks for adding the ref! Kk I’ll leave this one.

File: iati/about/models.py (hunk 1 above)
Comment (andylolz) [reply iati/about/models.py]: Kk – reverted f912bbd and fixed in 2ac2b11.

**Comment by andylolz** [comment]
```
Your code has been rated at 10.00/10
```

:tada: :tada: :tada:

**Comment by allthatilk** [comment]
Woop woop! Nice work @andylolz :shipit:

**Comment by dalepotter** [comment]
Great work @andylolz !

**Pull request merged**